feature-group-definitions/text-fragments.yml

```
@@ -0,0 +1,7 @@
+name: Text fragments
```

If we want to honor the precedent of MDN with "text fragments", maybe we could add some specificity to the end:

```suggestion
name: Text fragments in URLs
```

---

Hmm, what about our principle of shortest plausible name?

---

"URL references for text fragments"

Maybe it helps to think about a name for this feature when also thinking about how we used to only have links to ids before. I would have called that "URL references to fragment identifiers" and not just "Fragment identifiers", I assume.

---

Just "fragment identifiers" is what I would call that actually. I remember that from https://www.w3.org/TR/media-frags/ where "media fragments" were defined.

---

> Hmm, what about our principle of shortest plausible name?

I guess I thought the existing name wasn't completely credible. Over in https://github.com/web-platform-dx/web-features/pull/732#discussion_r1537332189 you seemed unimpressed with "text fragments" alone. 😄